e2e-test/src/test/features/sendemail/source/MacrosSendEmailAlert.feature

```diff
@@ -0,0 +1,67 @@
+    And Enter input plugin property: "referenceName" with value: "BQReferenceName"
+    And Enter input plugin property: "dataset" with value: "dataset"
+    And Enter input plugin property: "table" with value: "bqTargetTable"
+    And Click plugin property: "truncateTable"
+    And Click plugin property: "updateTableSchema"
+    And Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    And Click plugin property: "configureTab"
+    And Click plugin property: "pipelineAlertTab"
+    And Click plugin property: "createAlertButton"
+    And Select send email alert plugin
+    And Select dropdown plugin property: "emailRunCondition" with option value: "sendEmailCompletion"
+    And Click on the Macro button of Property: "emailSender" and set the value to: "sender"
+    And Click on the Macro button of Property: "emailRecipient" and set the value to: "recipients"
+    And Click on the Macro button of Property: "emailSubject" and set the value to: "subject"
+    And Click on the Macro button of Property: "emailMessage" and set the value in textarea: "message"
+    And Click on the Macro button of Property: "emailUsername" and set the value to: "username"
+    And Click on the Macro button of Property: "emailPassword" and set the value to: "password"
+    And Click on the Macro button of Property: "emailProtocol" and set the value to: "protocol"
+    And Click on the Macro button of Property: "emailHost" and set the value to: "host"
+    And Click on the Macro button of Property: "emailPort" and set the value to: "port"
+    And Click on the Macro button of Property: "emailIncludeWorkflowToken" and set the value to: "token"
+    And Validate send email plugin properties
+    And Click plugin property: "emailNextButton"
+    And Click plugin property: "emailConfirmButton"
+    And Click Pipeline Alert Save Button
+    And Save and Deploy Pipeline
+    And Run the Pipeline in Runtime
+    And Enter runtime argument value "sendEmailSender" for key "sender"
+    And Enter runtime argument value "sendEmailRecipients" for key "recipients"
+    And Enter runtime argument value "emailSubject" for key "subject"
+    And Enter runtime argument value "sendEmailMessage" for key "message"
+    And Enter runtime argument value "sendEmailUsername" for key "username"
+    And Enter runtime argument value "sendEmailPassword" for key "password"
+    And Enter runtime argument value "sendEmailProtocol" for key "protocol"
+    And Enter runtime argument value "sendEmailHost" for key "host"
+    And Enter runtime argument value "sendEmailPort" for key "port"
+    And Enter runtime argument value "sendEmailIncludeWorkflowToken" for key "token"
+    And Run the Pipeline in Runtime with runtime arguments
+    And Wait till pipeline is in running state
+    And Open and capture logs
+    And Verify the pipeline status is "Succeeded"
+    #And Verify email sent message in logs (Implement this step once PLUGIN-1255 is fixed.)
```

Comment:
I don't think it's expected that we would log a message when email is sent successfully. 
A RuntimeException is thrown in the failure case - does this show up in the pipeline logs when there is an error in sending email? https://github.com/cdapio/hydrator-plugins/blob/develop/core-plugins/src/main/java/io/cdap/plugin/batch/action/EmailAction.java#L154